submissions/smillims/memory-pair-game/js/script.js

```diff
@@ -1,89 +1,110 @@
 'use strict';
 
-const allCards = document.querySelectorAll('.flipper');
+const cards = document.querySelector('.cards');
 
-const cardPosition = 12;
-const allPairCards = 6;
-let currentPair = 0;
-let hasFlippedCard = false;
 let lockCard = false;
 let firstCard = '';
 let secondCard = '';
 
-function flipCards() {
-	if(lockCard){
-		return;
-	};
-	if (this === firstCard){
+const images = [
```

Comment:
its not images, it just images titles/names/alts, change naming for more corresponding, please